lib/DDG/Spice/DebVersion.pm

```diff
@@ -0,0 +1,26 @@
+# Get the version of a package in various debian repositories
```

Comment:
Mind putting this below the package declaration? The canonical approach is to set that as the `#ABSTRACT` or better yet the `description` for the metadata.